test/evptests.txt

```diff
@@ -1604,6 +1604,45 @@ Operation = ENCRYPT
 Plaintext = 000102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F20212223
 Ciphertext = A4DA23FCE6A5FFAA6D64AE9A0652A42CD161A34B65F9679F75C01F101F71276F15EF0D8D
 
+
+# ARIA test vectors from RFC5794
+Cipher = ARIA-128-ECB
+Key = 000102030405060708090a0b0c0d0e0f
+Operation = ENCRYPT
+Plaintext = 00112233445566778899aabbccddeeff
+Ciphertext = d718fbd6ab644c739da95f3be6451778
+
+Cipher = ARIA-128-ECB
+Key = 000102030405060708090a0b0c0d0e0f
+Operation = DECRYPT
+Plaintext = 00112233445566778899aabbccddeeff
+Ciphertext = d718fbd6ab644c739da95f3be6451778
```

Comment:
If you omit `Operation` then it tests both ways automatically. In other words you need only one copy of each vector. It's also easier to double-check against RFC text.